walkthroughs/eks/o11y-xray.md

```diff
@@ -13,15 +13,20 @@ $ aws iam attach-role-policy \
       --policy arn:aws:iam::aws:policy/AWSXRayDaemonWriteAccess
 ```
 
-Enable X-Ray tracing for the App Mesh data plane
+Enable X-Ray tracing for the App Mesh data plane. Optionally you can set the sampling rate by specifying the field `tracing.samplingRate=0.1` which corresponds to 10%. Also if you would like to write your own custom X-Ray tracing sampling rules then refer the example [howto-k8s-http2's template](../howto-k8s-http2/v1beta2/manifest.yaml.template) to see `xray-sampling-rules` configMap manually mounted on to the envoy proxy sidecar. Specifically in that given example we are trying to trace anything other than health check traffic. For more information on how to write this sampling-rules.json configMap please refer [AWS X-Ray Developer Guide](https://docs.aws.amazon.com/xray/latest/devguide/xray-sdk-go-configuration.html#xray-sdk-go-configuration-sampling).
```

Comment:
I like the existing example where it shows the bare minimum of getting xray to work, and the additional text in that section may be too much imo. I'd suggest moving this to a separate section where you can then elaborate on the ways users can configure the sampling rate.  we should also mention what the default sampling rate is.

Reply:
Sure, I accept this. Let me push a new revision asap.